bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/BootstrapLink.java

```diff
@@ -91,7 +91,7 @@ public BootstrapLink(final String id, final IModel<T> model, final Buttons.Type
 
         add(icon = newIcon("icon"));
         add(splitter = newSplitter("splitter"));
-        add(label = newLabel("label"));
+        add(label = newLabel("label", model));
```

Comment:
I'm not OK with this change. The model of the link as nothing to do with the label of the link.